lib/views/home/widgets/session_cards/widgets/card_widget.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:Medito/constants/constants.dart';
+import 'package:flutter/material.dart';
+
+class CardWidget extends StatelessWidget {
+  const CardWidget({super.key, this.tag, required this.title});
+  final String? tag;
+  final String title;
+  @override
+  Widget build(BuildContext context) {
+    var textTheme = Theme.of(context).textTheme;
+
+    var boxDecoration = BoxDecoration(
+      image: DecorationImage(
+        image: AssetImage(AssetConstants.dalle),
+        fit: BoxFit.cover,
+        colorFilter: ColorFilter.mode(
+          Colors.black.withOpacity(0.6),
+          BlendMode.dstATop,
+        ),
+      ),
+    );
+
+    return Padding(
+      padding: const EdgeInsets.only(left: 16),
+      child: Placeholder(
+        child: Container(
+          width: 154,
+          // decoration: boxDecoration,
```

Comment:
Comment

Reply:
Can we just ignore this comment for now? Because in the next PR, I will uncomment it.